### ISSUE TITLE
Add a driver for the CS1237 24 bit ADC and for the HX710 24 bit ADC.

### ISSUE THREAD

**Issue by robert-hh** [mcauser/awesome-micropython#153]
Hi Mike.
The CS1237 plays in the field of the HX711, but is told to be less noisy. My test did not show a huge difference, but that may be a problem of the test set-up. It's more flexible than the HX711, with gains of 1,2,64 and 128, and sampling rates of 10, 40, 640 and 1280.

https://github.com/robert-hh/CS1237

I made a variant of my HX711 driver for the HX710: https://github.com/robert-hh/hx710
My HX711 driver is at: https://github.com/robert-hh/hx711

The difference to the HX711 driver you listed already is the synchronization of the data reading with the data ready pulse on the data line, which will be created even if the data reading is in progress. If you do not synchronize the reading with that pulse, it may cause severely wrong values being retrieved.

Best Regards, Robert

**Comment by mcauser** [comment]
Thanks @robert-hh

Added CS1237 in a1fac96aeb91ee7f4a16046056e18a09159cdd7a
Added HX711 in bb7f876415d6788d0e05e15668ca2d5a034b1b6c
Added HX710 in 4fe836feecf618a0449a6642639d4140ba5ed778

I moved the the existing HX71x drivers to a new category `Sensors > Load Cell` and added yours there.
https://awesome-micropython.com/#load-cell

Added CS1237 to `IO > ADC`
https://awesome-micropython.com/#adc

Some of these drivers fit multiple categories. It's hard to pick the most appropriate. Let me know if you want anything recategorised.
